test/common/initModule.js

```diff
@@ -13,6 +13,8 @@ var database = require(path.join(dirname, '/helpers', 'database.js'));
 var genesisblock = require(path.join(dirname, '/genesisBlock.json'));
 var Logger = require(dirname + '/logger.js');
 var z_schema = require('../../helpers/z_schema.js');
+var cache = require('../../helpers/cache.js');
```

Comment:
Yeah, having `cache` and `Cache` variable might be confusing. I'll update it to something like `cacheHelper`.